master/static/srv/proto_get_trial_workloads.sql

```diff
@@ -38,18 +34,16 @@ checkpoints_vt AS (
         c.end_time,
         c.uuid,
         c.total_batches,
-        s.num_batches,
-        s.prior_batches_processed,
         c.resources
       FROM checkpoints c
         INNER JOIN steps s ON c.trial_id = s.trial_id
         AND c.total_batches = s.total_batches
       WHERE c.trial_id = $1
     ) AS r1
 )
-SELECT v.validation::jsonb - 'total_batches' AS validation,
-  t.training::jsonb - 'total_batches' AS training,
-  c.checkpoint::jsonb - 'total_batches' AS checkpoint
```

Comment:
lol, it was really going out of the way to remove `total_batches`!

Reply:
lol seriously